compiler/luci/partition/src/ConnectNode.test.h

```diff
@@ -105,6 +105,23 @@ class ConnectionTestHelper
     }
   }
 
+  /**
+   * @note prepare_inputs_miss is for negative testing
+   */
+  template <unsigned N> void prepare_inputs_miss(TestIsOGraph<N> *isograph)
+  {
+    assert(N == isograph->num_inputs());
+
+    for (uint32_t i = 0; i < N; ++i)
+    {
+      auto *input = _graph_clone->nodes()->create<luci::CircleInput>();
+      luci::copy_common_attributes(isograph->input(i), input);
+      if (i != 0)
+        _clonectx.emplace(isograph->input(i), input);
+      _inputs.push_back(input);
+    }
+  }
```

Comment:
I wonder there's an any difference between normal one. Above inputs are invalid?

Reply:
Yup. we remove first input so the connection to input SHOULD fail. this is the purpose.

Reply:
Oh, I missed that part. Now I got it:)